_source/lib/collections/_helpers.coffee

```diff
@@ -1,34 +1,45 @@
-
 Apollos.schemas = {}
 
-Apollos.generateSchema = (name, schema) ->
+Apollos.generateSchema = (name, excludeTrackingFields, schema) ->
```

Comment:
@benwiley there is a version of this in the refactor-package branch that puts the tracking fields bool at the end of the function arguments. It was done that way as to not be a breaking change for schema generation. I would prefer it to be at the end of the arguments like an opts trigger